Source/Core/Core/PowerPC/MMU.h

```diff
@@ -13,31 +13,53 @@
 
 namespace PowerPC
 {
+enum TranslateMode
```

Comment:
If an enum must be used, please use a strongly typed enum to prevent the identifiers from leaking into the PowerPC namespace and to also enforce explicit conversions in the interface.

```suggestion
enum class TranslateMode
{
  Host,
  Never,
  Always,
};
```